nix/hubris.nix

```diff
@@ -32,8 +32,6 @@ stdenv.mkDerivation rec {
     # xtask uses this variable to find the hubris root, expecting to be run as `cargo xtask dist ...`.  Since we are invoking directly we need to manually set this path.
     export CARGO_MANIFEST_DIR=$(pwd)/build/xtask
 
-    # format first so we don't go through the lengthy build only to fail for formatting
```

Comment:
Are we happy to lose the behaviour described in this comment? Might be worth mentioning in the commit msg if so.

Also:
s/parody/parity